pom.xml

```diff
@@ -52,6 +53,12 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <!-- Add the specific version of SnakeYAML that doesn't have the vulnerabilities -->
```

Comment:
for remove  vulnerabilities

Reply:
coming  vulnerabilities warning on this dependencies 
<dependency>
            <groupId>org.springframework.boot</groupId>
            <artifactId>spring-boot-starter-web</artifactId>
        </dependency>